src/components/ItemEditorPage/RightPanel/RightPanel.container.ts

```diff
@@ -14,12 +14,13 @@ const mapState = (state: RootState): MapStateProps => {
   const selectedItemId = getSelectedItemId(state) || ''
   const selectedItem = getItem(state, selectedItemId)
   const address = getAddress(state) || ''
-
+  const collection = selectedItemId && selectedItem && selectedItem.collectionId ? getCollection(state, selectedItem.collectionId) : null
   return {
     address,
-    collection: selectedItemId && selectedItem && selectedItem.collectionId ? getCollection(state, selectedItem.collectionId) : null,
+    collection,
     selectedItem,
     selectedItemId,
+    canEditSelectedItem: !!selectedItem && !!collection && hasViewAndEditRights(state, address, collection),
```

Comment:
Is it considering only the case which the collection exists? We'll be leaving behind the cases where we want to edit a single item that doesn't belong to a collection.

Reply:
Oh yeah, you're right! Updated to cover those cases as well!